Source/relay/TourGuide/Paths/Bad Moon.ash

```diff
@@ -71,9 +71,9 @@ static
         __static_bad_moon_adventures.listAppend(BadMoonAdventureMake(5, $location[Cobb's Knob Kitchens], "STAT2", "+40 myst, -50% familiar weight", "", false));
         __static_bad_moon_adventures.listAppend(BadMoonAdventureMake(6, $location[Cobb's Knob Harem], "STAT2", "+40 moxie, -50% familiar weight", "", false));
         __static_bad_moon_adventures.listAppend(BadMoonAdventureMake(7, $location[Frat House], "STAT3", "+50% muscle, -50% myst", "", false));
-        __static_bad_moon_adventures.listAppend(BadMoonAdventureMake(8, $location[Frat House In Disguise], "STAT3", "+50% muscle, -50% moxie", "", false));
+        __static_bad_moon_adventures.listAppend(BadMoonAdventureMake(8, $location[Frat House (frat disguise)], "STAT3", "+50% muscle, -50% moxie", "", false));
```

Comment:
this is not quite the right change. it isn't "Frat House (frat disguise)", it's "Frat House (in disguise)". see screenshot below.

![image](https://user-images.githubusercontent.com/8014761/183573195-f1a03c67-cda1-46a0-89f1-00dfaba6b642.png)

please change this for both frat/hippy camp in bad moon.ash